private/event.nim

```diff
@@ -55,7 +55,7 @@ var wxEVT_USER_FIRST* {.importcpp: "wxEVT_USER_FIRST", header: wxh.}: WxEventTyp
 
 type 
   WxEventCategory* {.size: sizeof(cint), 
-                     importcpp: "wxEventConnectionRef::wxEventCategory", 
+                     importcpp: "wxEventCategory", 

```

Comment:
Hmm, why did this change? Do you have a minimal example I can reproduce this error with?